include/pear/Mail/mimeDecode.php

```diff
@@ -369,6 +369,7 @@ function _decode($headers, $body, $default_ctype = 'text/plain')
 						$return->body = ($this->_decode_bodies ? $this->_decodeBody($body, $encoding) : $body);
 					}
                     $obj = new Mail_mimeDecode($body);
+                    $return->body = $body;
```

Comment:
shoule we say if body is not set? This is because body can potentially be set in like `369`.
```php
if (!isset($return->body))
    $return->body = $body;
```